HyphaApp/hypha
# Submission data on determination and review form page

Fixes #3971

## Test Steps

- Add or edit a determination (e.g. `/apply/submissions/2/determination/2/edit/`)
- Click "Toggle submission" button in the header to see the submission.
- Same thing for reviews
